node_modules/oae-admin/manageuser/manageuser.html

```diff
@@ -46,10 +46,16 @@ <h3 id="manageuser-modal-title"><!-- --></h3>
                 </form>
 
                 <form id="manageuser-actions-form" class="tab-pane form-horizontal" role="form">
```

Comment:
You could even put them in 2 forms and hide/show the forms. The advantage is that you can then bind to the submit action of both forms, preventing to have bind to the button click.